python/pylibcugraph/pylibcugraph/graphs.pyx

```diff
@@ -125,13 +146,16 @@ cdef class SGGraph(_GPUGraph):
                   GraphProperties graph_properties,
                   src_or_offset_array,
                   dst_or_index_array,
+                  vertices_array=None,
```

Comment:
Can you move this to a position after `input_array_format` so calling this with position-only params still works?  Otherwise, this requires users to use kwarg params which would be a breaking change.

Reply:
Right! I had some tests failing because of this so I passed argument by keyword. I will also preserve the order (position) as you suggested.